### PR TITLE
Upgrade tonics deps

### PR DESCRIPTION
**Pull request by aoudiamoncef**

### CODE REVIEW

File: src/massa.api.v1.rs [hunk 1]
```diff
@@ -1177,7 +1177,11 @@ pub mod private_service_server {
                         ) -> Self::Future {
                             let inner = Arc::clone(&self.0);
                             let fut = async move {
-                                (*inner).add_to_bootstrap_blacklist(request).await
+                                <T as PrivateService>::add_to_bootstrap_blacklist(
+                                        &inner,
+                                        request,
+                                    )
+                                    .await
                             };
```
Comment (damip) [inline src/massa.api.v1.rs:1185]: why not just

```
let fut = add_to_bootstrap_blacklist(&inner,  request);
```

Comment (aoudiamoncef) [reply src/massa.api.v1.rs:1185]: Autogenerated stuff

Comment (damip) [reply src/massa.api.v1.rs:1185]: oki

**Review by damip — APPROVED** [review]

**Comment by aoudiamoncef** [comment]
@AurelienFT related to https://github.com/massalabs/massa-sc-runtime/pull/308

**Review by AurelienFT — APPROVED** [review]

**Pull request merged**